graph4nlp/pytorch/data/dataset.py

```diff
@@ -311,6 +311,7 @@ def __init__(
         for_inference=False,
         reused_vocab_model=None,
         nlp_processor_args=None,
+        init_edge_vocab=False,
```

Comment:
init_edge_vocab is not clear. It covers 1.build edge vocab, 2. an indicator to use heterogeneous graph. 
TODO: discuss

Reply:
1. build_edge_vocab: bool
2. is_hetero: bool